core/tests/Test.Microsoft.Identity.Core.Unit/Test.Microsoft.Identity.Core.Unit.csproj

```diff
@@ -79,7 +79,6 @@
     <PackageReference Include="MSTest.TestFramework" Version="1.3.1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\msal\src\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.Identity.Core.csproj" />
```

Comment:
><ProjectReference Include="..\..\src\Microsoft.Identity.Core.csproj" /> [](start = 4, length = 71)

Not sure this line makes sense to me. Does that still exist as a separate project? I thought it was now reduced to a part of each of the libraries?